Source/src/importers/PrefabImporter.cpp

```diff
@@ -109,10 +91,60 @@ Hachiko::UID Hachiko::PrefabImporter::CreatePrefabAsset(const char* name, GameOb
 
     root->CollectObjectsAndComponents(object_collector, component_collector);
     root->SavePrefabReferences(prefab->prefab_data, object_collector, component_collector);
+}
+
+void Hachiko::PrefabImporter::LoadScriptReferences(ResourcePrefab* prefab, GameObject* root)
+{
+    // Wire script references by indexes
+    std::vector<const GameObject*> object_collector = std::vector<const GameObject*>();
```

Comment:
to initialize this you don't need to assign an empty vector, using {} would be enough